src/lexers/javascript-lexer.js

```diff
@@ -14,8 +18,21 @@ export default class JavascriptLexer extends BaseLexer {
   extract(content) {
     const that = this
 
+    var localAcorn = acorn;
```

Comment:
Please use `let` instead of `var`

Reply:
Also, why localAcorn? Don't the inject functions take the input object, modify it, and pass it back? In other words, at line 33, is this true: `localAcorn === acorn`?

Reply:
@MynockSpit I was not keen on modifying the imported acorn object, which is read-only. The stage3 injector `injectAcornStage3` needs the result to be assigned (line 25). That's why a local variable is created. 

In line 33 we have no guarantee that the object reference will always be the same. A sentence like `return Object.assign({}, acorn, ...)` inside the stage3 injector would break the equality.